apps/nextjs/src/components/icons/picker/icon-picker.tsx

```diff
@@ -35,33 +35,48 @@ export const IconPicker = ({ initialValue, onChange, error, onFocus, onBlur }: I
 
   const groups = notNullableData.map((group) => {
     const options = group.icons.map((item) => (
-      <Combobox.Option value={item.url} key={item.id}>
-        <Group>
-          <Image src={item.url} w={20} h={20} />
-          <Text>{item.name}</Text>
-        </Group>
-      </Combobox.Option>
+      <Card
+        onClick={() => {
+          const value = item.url;
+          setValue(value);
+          setPreviewUrl(value);
+          setSearch(value);
+          onChange(value);
```

Comment:
previewUrl and value seem to have the same values.

Reply:
That is not true. See ``onChange``